diff --git a/node_modules/spawn-command/lib/spawn-command.js b/node_modules/spawn-command/lib/spawn-command.js
index 24d56ab..f23494c 100644
--- a/node_modules/spawn-command/lib/spawn-command.js
+++ b/node_modules/spawn-command/lib/spawn-command.js
@@ -1,4 +1,3 @@
-var util = require('util');
 var spawn = require('child_process').spawn;
 
 module.exports = function (command, options) {
@@ -6,7 +5,7 @@ module.exports = function (command, options) {
   if (process.platform === 'win32') {
     file = 'cmd.exe';
     args = ['/s', '/c', '"' + command + '"'];
-    options = util._extend({}, options);
+    options = Object.assign({}, options);
     options.windowsVerbatimArguments = true;
   }
   else {
